### PR TITLE
Add Matija Marohnić

### PR DESCRIPTION
**Pull request by silvenon**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -51,6 +51,7 @@
 - [Hugo Giraudel](https://github.com/hugogiraudel/ama) - CSS goblin, Sass hacker, margin psycho.
 - [Will Binns-Smith](https://github.com/wbinnssmith/ama) - JavaScript at Bitbucket. Makes internet things.
 - [Lauri Tervonen](https://github.com/mollikka/ama) - CS & SE university student in Finland.
+- [Matija Marohnić](https://github.com/silvenon/ama) - From modding Warcraft to frontend design/development and nerdiness for Node.js build tools
```
Comment (arthurvr) [inline readme.md]: Misses a period, fixed that for you :wink: Welcome!

Comment (silvenon) [inline readme.md]: Thanks :wink:

**Comment by sindresorhus** [comment]
Hey. Want to ask you a favor. Would be cool if you could ask some questions on the other AMAs in the list (not my AMA). I think AMAs are a great opportunity to learn from each other. Asking a question is also a good way to encourage receiving one ;)

For inspiration of what to ask, you could take a look at the questions in some other AMAs, like:
https://github.com/passy/ama/issues?q=
https://github.com/sebmck/ama/issues?q=
https://github.com/wesbos/ama/issues?q=
https://github.com/sindresorhus/ama/issues?q=

**Comment by silvenon** [comment]
Sure! I participated far less than I would like to anyway.

**Pull request closed without merging**